lua/telescope/_extensions/zoxide/list.lua

```diff
@@ -60,20 +60,14 @@ local fuzzy_with_z_score_bias = function(opts)
 end
 
 local entry_maker = function(item)
-  local items = vim.split(string.gsub(item, '^%s*(.-)%s*$', '%1'), " ")
-  local score = 0
-  local item_path = item
-
-  if #items > 1 then
-    score = tonumber(items[1])
-    item_path = items[2]
-  end
+  local trimmed = string.gsub(item, '^%s*(.-)%s*$', '%1')
+  local item_path = string.gsub(trimmed, '^[^%s]* (.*)$', '%1')
+  local score = string.gsub(trimmed, '^([^%s]*) .*$', '%1')
```

Comment:
N00b question. The idea as that the `score` was facilitated as a `number` type, so anyone can interpret it as a number (sort, math, go crazy).

I am a bit out of Lua at the moment, could this cause any problems for the existing user-base?

Reply:
Fair push, can cast